dynamic_gantt_planning/i18n/nl_BE.po

```diff
@@ -0,0 +1,86 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
```

Comment:
@bizzappdevteam @bizzappdev please rename the file to `nl.po`, this will cover both the Netherlands & Belgium then as `nl_BE` inherits from `nl` if there is no term